bindings/c/test/mako/ddsketch.hpp

```diff
@@ -89,6 +90,13 @@ class DDSketchBase {
 		} else {
 			int index = static_cast<Impl*>(this)->getIndex(sample);
 			assert(index >= 0 && index < int(buckets.size()));
+			if (buckets[index] == std::numeric_limits<uint32_t>::max()) {
+				std::cerr << "Terminating. "
+				          << "Maximum threshold for statistics collection hit "
+				          << "has hit maximum size." << std::endl;
+				std::cerr << "Consider increasing --sampling to a smaller value.";
```

Comment:
`Consider increasing --sampling to a smaller value.` is a very strange sentence. Perhaps, "Consider increasing --sampling to sample less frequently".

We should really change how sampling rate is defined...so confusing.

Reply:
That sentence does look bizarre.  Sampling is defined to be the amount of operations that occur between samples. So if you were to decrease the sampling _value_, you would be collecting more samples overall

Reply:
I updated the message.